about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tesfaye-capstone-project</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+
+</head>
+<body>
+    <div class="header-container">
+        <div class="header-nav">
+                <ul>
+                <li><a href="#"><img src="images/icons/facebook.png" /> </a></li> 
+                <li><a href="#"><img src="images/icons/twitter.png" /> </a></li>
+                <li><a href="#">English </a></li>
+                <li><a href="#">My Page </a></li>
+                <li><a href="#">Logout </a></li>
+                </ul>
+        </div>
+    </div> 
+    <header>
+    <div class="nav-container">
```

Comment:
- [ ] On the Mobile version, the hamburger button for the navbar doesn't work. Kindly make sure it works so we can use the funtionalties added there.